features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/data/api.js

```diff
@@ -419,6 +419,25 @@ class API extends Resource {
         }
     }
 
+       /**
+     * Get the graphQL schema of an API
+     * @param id {String} UUID of the API in which the swagger is needed
+     * @param callback {function} Function which needs to be called upon success of the API deletion
```

Comment:
is this description about callback correct?

Reply:
Fixed with 2a6fbdd3ac9ce54ca7bc94a45328a41826836a80